src/zope/schema/_field.py

```diff
@@ -615,10 +616,21 @@ class Object(Field):
     __doc__ = IObject.__doc__
 
     def __init__(self, schema, **kw):
+        """
+        Object(schema, validate_invariants=True, **kwargs)
+
+        Create an `~.IObject` field. The keyword arguments are as for `~.Field`.
+
+        .. versionchanged:: 4.6.0
```

Comment:
That, plus @tseaver recently told me that he holds off on bumping the numbers in CHANGES and setup.py until release time (presumably with zest.releaser) to avoid forgetting (in the specific PR, I had changed it one place but not the other), so I was giving that a try.

Reply:
Ha!  I always bump as soon as something semver-ish happens, for the same reason: to avoid forgetting!